Fix specs, travis build

severinraez opened the pull request:
I looked at the code in preparation for an attempt on https://github.com/salsify/delayed_job_worker_pool/pull/3 . Some side products:

* I've fixed the build on travis, needed to add some require's
* I've added Ruby 2.5.5 and pinned dependencies to keep the build green (Rails 6 and Sprockets 4 will both break it)
* Red rebuild of your current master: https://travis-ci.org/severinraez/delayed_job_worker_pool/builds/596486724
* Build of this PR: https://travis-ci.org/severinraez/delayed_job_worker_pool/builds/596494839
* Ruby 2.1 and 2.0 are failing on both master and my branch. To fix 2.1, we'd need another pin: `spec.add_development_dependency 'sqlite3', '>= 1.3', '< 1.4'`. Didn't look at 2.0. With those versions now being EOL, what do you think about dropping support for them to keep things simple?

jturkel commented on .travis.yml:
```diff
@@ -1,5 +1,6 @@
 language: ruby
 rvm:
+  - 2.5.5
   - 2.4.0
```
Mind bumping this to the latest 2.4 release while you’re in here?

jturkel commented on .travis.yml:
```diff
@@ -1,5 +1,6 @@
 language: ruby
 rvm:
+  - 2.5.5
```
Any reason not to test against the latest 2.5 patch release (2.5.7)?

severinraez commented:
@jturkel at last... :)

* travis: Removed all EOL rubies (up to and including 2.4), added newer rubies
* travis: Using latest patch releases
* Bumped bundler to 2.1

Please merge this if it looks good. 

What do you think about upgrading the dummy app to rails 6? This would enable us to update the dev dependencies in the gemspec and could be done in another PR. For me personally, the next goal is to work on https://github.com/salsify/delayed_job_worker_pool/pull/3

jturkel commented:
Thanks @severinraez! Upgrading the dummy app to Rails 6.0 sounds good to me.
